### PR TITLE
Wishlist functionality

### PR DESCRIPTION
**Pull request by sameep-s**
Wishlist functionality added:

- Added items into wish-list by clicking on the heart icon.
- Remove items by clicking on heart icon twice.
- Items gets added in the wishlist.

Screenshot (123)
 
Screenshot (124)

Screenshot (125)

### CODE REVIEW

**Review by fahd23 — APPROVED** [review]
Code looks clean and well structured

File: src/util_reducers/wishlist-reducer.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+export const WishlistReducer = (state_Wishlist, action_Wishlist) => {
+
+    const { wishlist } = state_Wishlist;
+    console.log(`wishlist:`, wishlist);
```
Comment (tusharkandpal) [inline src/util_reducers/wishlist-reducer.js:4]: Can remove this logger

**Review by tusharkandpal — APPROVED** [review]
why aren't there any api calls for wishlist ?

File: src/util_reducers/wishlist-reducer.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+    switch (action_Wishlist.type) {
+        case "ADD_TO_WISHLIST":
+            if (wishlist.filter(item => item._id === action_Wishlist.payload._id).length === 1)
+                return { ...state_Wishlist, wishlist: [...wishlist.filter(item => item._id !== action_Wishlist.payload._id)] }
+
+            return { ...state_Wishlist, wishlist: [...wishlist, action_Wishlist.payload] }
+
```
Comment (tusharkandpal) [inline src/util_reducers/wishlist-reducer.js:12]: can make use of ternary operator here

File: src/util_reducers/wishlist-reducer.js (hunk 1 above)
Comment (sameep-s) [reply src/util_reducers/wishlist-reducer.js:4]: sure i will.

File: src/util_reducers/wishlist-reducer.js (hunk 2 above)
Comment (sameep-s) [reply src/util_reducers/wishlist-reducer.js:12]: Sure i will take it into consideration.

**Pull request merged**